shared/utils/urls.ts

```diff
@@ -86,7 +90,8 @@ export function sanitizeUrl(url: string | null | undefined) {
     !isUrl(url) &&
     !url.startsWith("/") &&
     !url.startsWith("#") &&
-    !url.startsWith("mailto:")
+    !url.startsWith("mailto:") &&
+    !url.includes("://")
```

Comment:
To avoid breaking the protocols if any is specified

Reply:
Hey @iifawzi this change basically nullifies the sanitize method and allows any protocol through. I suggest that we maintain a blocklist of protocols starting with `javascript` and `file` instead

Reply:
Hey @tommoor, thank you for your time reviewing this, to make sure I got it right, you prefer to have a list of  `notAllowedProtocols` for example,  and it can initially include `javascript://` and `file://`? so we can avoid these protocols from being sanitized, right?

Reply:
Yes, `blockedProtocols` must be removed, in this case prepended with `https` as before is the way to make them "safe".

Reply:
That's really cool, I got it, changes have been committed. 

I've used regex to detect the protocol, a simpler approach can be as simple as

```js
 blockedProtocols.some(protocol => url.includes(protocol))
```

let me know if it could be improved, or if i'm missing anything